lib/faker/default/company.rb

```diff
@@ -5,57 +5,165 @@ class Company < Base
     flexible :company
 
     class << self
+      ##
+      # Produces a company name.
+      #
+      # @return [String]
+      #
+      # @example
+      #   Faker::Company.name #=> "Roberts Inc"
+      #
+      # @faker.version 1.6.0
       def name
         parse('company.name')
       end
 
+      ##
+      # Produces a company suffix.
+      #
+      # @return [String]
+      #
+      # @example
+      #   Faker::Company.suffix #=> "LLC"
+      #
+      # @faker.version 1.6.0
       def suffix
         fetch('company.suffix')
       end
 
+      ##
+      # Produces a company industry.
+      #
+      # @return [String]
+      #
+      # @example
+      #   Faker::Company.industry #=> "Food & Beverages"
+      #
+      # @faker.version 1.6.0
       def industry
         fetch('company.industry')
       end
 
-      # Generate a buzzword-laden catch phrase.
+      ##
+      # Produces a company catch phrase.
+      #
+      # @return [String]
+      #
+      # @example
+      #   Faker::Company.catch_phrase #=> "Grass-roots grid-enabled portal"
+      #
+      # @faker.version 1.6.0
       def catch_phrase
         translate('faker.company.buzzwords').collect { |list| sample(list) }.join(' ')
       end
 
+      ##
+      # Produces a company buzzword.
+      #
+      # @return [String]
+      #
+      # @example
+      #   Faker::Company.buzzword #=> "flexibility"
+      #
+      # @faker.version 1.8.7
       def buzzword
         sample(translate('faker.company.buzzwords').flatten)
       end
 
+      ##
+      # Produces a company bs.
+      #
+      # @return [String]
+      #
+      # @example
+      #   Faker::Company.bs #=> "empower customized functionalities"
+      #
+      # @faker.version 1.6.0
+      #
       # When a straight answer won't do, BS to the rescue!
       def bs
         translate('faker.company.bs').collect { |list| sample(list) }.join(' ')
       end
 
+      ##
+      # Produces a company ein.
```

Comment:
Should clarify what an EIN is.
```suggestion
      # Produces a company EIN (Employer Identification Number).
```

Reply:
@connorshea 
Thank you for your suggestion.
I fixed it.